src/lib/FunctionAssertion.ts

```diff
@@ -29,25 +30,41 @@ export class FunctionAssertion<T extends AnyFunction> extends Assertion<T> {
   }
 
   /**
-   * Check if the function throws anything when called.
+   * Check if the function throws when called. Optionally, you can check that
+   * the thrown error is strictly equal an `Error` instance by passing it as a
```

Comment:
I found a typo here: 

```suggestion
   * the thrown error is strictly equal to an `Error` instance by passing it as a
```